client/src/components/Questions/qna.css

```diff
@@ -0,0 +1,355 @@
+strong {
+  font-weight: bold;
+}
+#qna {
+	border: 2px solid black;
+	border-radius: 5px;
+	margin: 5px 5% 5px 5%;
+
+}
+.qna-title {
+  color: black;
```

Comment:
nit: At some point in the future (maybe not right now) we should make sure our CSS formatting / spacing is consistent. It seems like ESLint isn't checking our CSS files, but maybe there is a way to configure it to do so.